trade-finance-manager-ui/templates/case/underwriting/pricing-and-risk/edit-facility-risk-profile/edit-facility-risk-profile.njk

```diff
@@ -5,6 +5,8 @@
 
 {% extends "index.njk" %}
 
+{% block pageTitle %}What’s the risk profile for this facility?{% endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.